client/me/help/help-contact/index.jsx

```diff
@@ -581,7 +587,9 @@ const HelpContact = React.createClass( {
 		const olarkReadyOrTimedOut = olark.isOlarkReady || this.props.olarkTimedOut;
 		const ticketReadyOrError = ticketSupportConfigurationReady || null != ticketSupportRequestError;
 
-		return ! sitesInitialized || ! ticketReadyOrError || ! olarkReadyOrTimedOut;
+		const waitingOnDirectly = this.shouldUseDirectly() && this.props.directlyStatus !== DIRECTLY_STATUS_READY;
```

Comment:
Maybe add another selector for this called `isDirectlyReady` so we don't need to check for these statuses.